### PR TITLE
Enable emulated mode in Openshift 4.18

### PR DESCRIPTION
**Pull request by harche**
This PR adds support for running emulated mode in OCP 4.18.

### CODE REVIEW

File: docs/instaslice-emulator-openshift-4.18.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+   ```bash
+   kubectl describe instaslice
+   ```
+
+4. **Submit an emulator pod**:
+
+   Use the following command to submit an emulator pod:
+
+   ```bash
+   kubectl apply -f samples/emulator-pod.yaml
```
Comment (rphillips) [inline docs/instaslice-emulator-openshift-4.18.md:125]: nit: since we are working against openshift, steps 2,3,4 should probably use `oc`.

Comment (harche) [reply docs/instaslice-emulator-openshift-4.18.md]: updated, thanks.

File: config/rbac/openshift_role_binding.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```
[the remark this answers is not in the document]
Comment (harche) [reply config/rbac/openshift_role_binding.yaml]: Thanks, fixed.

**Comment by harche** [comment]
> /hold for #131

/hold cancel

**Pull request merged**